spec/lib/network/port_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'manageiq/network/discovery'
+require 'ostruct'
+require 'benchmark'
+
+RSpec.describe ManageIQ::Network::Port do
+  before(:each) do
+    @ost = OpenStruct.new
+    @ost.ipaddr = "192.168.252.2" # yoda (ESX)
```

Comment:
What is this comment?  Also, can you use a documentation IP address? https://en.wikipedia.org/wiki/IPv4#Special-use_addresses